src/common/collection.c

```diff
@@ -1458,7 +1458,7 @@ static gchar *get_query_string(const dt_collection_properties_t property, const
         query = dt_util_dstrcat(query, "(film_id IN (SELECT id FROM main.film_rolls WHERE folder LIKE '%s%%'))",
                                 escaped_text);
       else
-        query = dt_util_dstrcat(query, "(film_id IN (SELECT id FROM main.film_rolls WHERE folder LIKE '%s'))",
+        query = dt_util_dstrcat(query, "(film_id IN (SELECT id FROM main.film_rolls WHERE folder = '%s'))",
```

Comment:
this will break a seach like `%e%` for example.

Reply:
not the same: the old one is case insensitive, not the second one.